src/main/java/org/broadinstitute/consent/http/models/Dataset.java

```diff
@@ -297,6 +304,59 @@ public void setSharingPlanDocumentName(String sharingPlanDocumentName) {
         this.sharingPlanDocumentName = sharingPlanDocumentName;
     }
 
+    /**
+     * Checks if the Dataset matches a raw search query. Searches on all dataset properties
+     * and some data use properties.
+     *
+     * @param query Raw string query
+     * @return if the Dataset matched query
+     */
+    public boolean isStringMatch(@NonNull String query) {
+        String lowerCaseQuery = query.toLowerCase();
+        List<String> queryTerms = new ArrayList<>(List.of(lowerCaseQuery.split("\\s+")));
```

Comment:
No need for the extra ArrayList construction ... 
```suggestion
        List<String> queryTerms = List.of(lowerCaseQuery.split("\\s+"));
```

Reply:
good catch - originally had this because `List.of` is immutable & I was adding things later, but I removed that 👍